File: java/11285.java
import java.io.*;
import java.util.*;
import java.lang.Math;
class Solution
{
	public static void main(String args[]) throws Exception
	{
		BufferedReader br=new BufferedReader(new InputStreamReader(System.in));
        int T=Integer.parseInt(br.readLine());
        for(int test_case = 1; test_case <= T; test_case++)
		{
			float N=Integer.parseInt(br.readLine());
			int ans=0;
            for(int ccase=1;ccase<=N;ccase++)
            {
                StringTokenizer st=new StringTokenizer(br.readLine()," ");
             	double x=Double.parseDouble(st.nextToken());
                double y=Double.parseDouble(st.nextToken());
                double cir=Math.pow((Math.pow(x,2)+Math.pow(y,2)),0.5);
                int tmp = 10-(int)(cir/20);
                if (cir%20==0){
                    tmp+=1;
                }
                if(cir>200){
                    tmp=0;
                }
                if(x==0&&y==0){
                    tmp=10;
                }
                ans+=tmp;
                
            }
            System.out.printf("#%d %d\n",test_case,ans);
        }
	}
}

File: python/11285.py
ans=[]
for m in range(int(input())):
    tmp=0
    N=int(input())
    loc=[]
    checker=[i*i for i in range(0,201,20)]
    for i in range(N):
        x,y=map(int,input().split())
        cir=((x*x+y*y))
        for j in range(1,11):
            if cir<=checker[j]:
                tmp+=11-j
                break
    ans.append("#"+str(m+1)+" "+str(tmp))
print(*ans,sep="\n")